broken-site-reporting/README.md

```diff
@@ -14,6 +14,10 @@ Test suite specific fields:
 - `wasUpgraded` - bool - if request was upgraded to HTTPS by us or not
 - `category` - string - user picked breakage category e.g. 'content', 'images', 'paywall', 'login'
 - `blockedTrackers` - array of strings - array of hostnames of trackers that were blocked
+- `noActionRequests` - array of strings - array of hostnames of requests that had `none` action
+- `adAttributionRequests` - array of strings - array of hostnames of requests that had `ad-attribution` action
+- `ignoredByUserRequests` - array of strings - array of hostnames of requests that had `ignore-user` action
```

Comment:
none/ad-attribution/ignore-user - those are extension specific names, I don't think we should use them here, but rather explain what they mean